dll/opengl/glu32/src/libnurbs/internals/quilt.cc

```diff
@@ -62,7 +62,8 @@ Quilt::deleteMe( Pool& p )
 {
     for( Quiltspec *q=qspec; q != eqspec; q++ ) {
 #if 1
-	if( q->breakpoints) delete[] q->breakpoints;  q->breakpoints = 0;
+	if( q->breakpoints) delete[] q->breakpoints;
+    q->breakpoints = 0;
```

Comment:
This line is inside for statement, shouldn't it has the same indentation as if(...)?

Reply:
oh, that''s a tab vs spaces issue. I'll fix it.